src/components/TodoList.tsx

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+
+import { useContext } from 'react';
+import { TodosContext } from './TodosContext';
+import { TodoItem } from './TodoItem';
+import { FilterOption } from '../types/FilterOption';
+
+export const TodoList: React.FC = () => {
+  const { todos, setTodos, filterOption } = useContext(TodosContext);
+
+  const visiableTodo = todos.filter(todo => {
+    if (filterOption === FilterOption.All) {
+      return true;
+    }
+
+    if (filterOption === FilterOption.Active) {
+      return !todo.completed;
+    }
+
```

Comment:
Use [`switch case`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch) in cases where you constantly compare one value with others